frontend/catalyst/jax_primitives.py

```diff
@@ -25,21 +26,51 @@
 from jaxlib.mlir.dialects._func_ops_gen import CallOp
 from jaxlib.mlir.dialects._mhlo_ops_gen import ConstantOp, ConvertOp
 
-from catalyst.python_bindings._arith_ops_gen import IndexCastOp
-from catalyst.python_bindings._tensor_ops_gen import ExtractOp as TensorExtractOp, FromElementsOp
-from catalyst.python_bindings._scf_ops_gen import IfOp, ConditionOp, ForOp, WhileOp, YieldOp
-from catalyst.python_bindings import AllocOp, ExtractOp, InsertOp, DeallocOp
-from catalyst.python_bindings import CustomOp, MultiRZOp, QubitUnitaryOp, MeasureOp
-from catalyst.python_bindings import SampleOp, CountsOp, ExpvalOp, VarianceOp, ProbsOp, StateOp
-from catalyst.python_bindings import GradOp
-from catalyst.python_bindings import (
+from catalyst._configuration import INSTALLED
+from catalyst.utils.calculate_grad_shape import calculate_grad_shape, Signature
+
+default_bindings_path = os.path.join(
+    os.path.dirname(__file__), "../../mlir/build/python_packages/quantum"
+)
+if not INSTALLED and os.path.exists(default_bindings_path):  # pragma: no cover
+    import sys
+
+    sys.path.insert(0, default_bindings_path)
```

Comment:
I think we might want to move this to the top-level `__init__.py`, we have the same lines in `compilation_pipelines.py` as well.

(Also prevents us from having to disable the wrong import order error several times here.)